test/test_speed_v_torch.py

```diff
@@ -139,7 +139,7 @@ def f(a, b): return a @ b
     helper_test_generic_square('gemm', 4096, f, f)
   def test_large_conv_1x1(self): helper_test_conv(bs=32, in_chans=128, out_chans=128, kernel_size=1, img_size_y=128, img_size_x=128)
   def test_large_conv_3x3(self): helper_test_conv(bs=4, in_chans=128, out_chans=128, kernel_size=3, img_size_y=130, img_size_x=130)
-  def test_large_conv_5x5(self): helper_test_conv(bs=4, in_chans=128, out_chans=128, kernel_size=5, img_size_y=130, img_size_x=130)
+  def test_large_conv_5x5(self): helper_test_conv(bs=4, in_chans=128, out_chans=128, kernel_size=5, img_size_y=132, img_size_x=132)
```

Comment:
Why did this change?

Reply:
I made this change to have an 128x128 output (the conv is without padding). We need the width to be divisible by 8 for tensor cores. Otherwise, it will just do a 2D-conv without tensor cores.

We should probably add some general padding and cropping logic in future to round up dimensions to the next "nice number" so that optimizations can kick in.